code/modules/atmospherics/machinery/components/fusion/hypertorus.dm

```diff
@@ -1021,87 +1021,87 @@
 		if(power_output)
 			switch(power_level)
 				if(1)
-					var/scaled_production = clamp(heat_output * 1e-2, 0, fuel_consumption) * delta_time
-					moderator_internal.gases[/datum/gas/carbon_dioxide][MOLES] += scaled_production * 0.95
-					moderator_internal.gases[/datum/gas/water_vapor][MOLES] += scaled_production *0.75
+					var/scaled_production = clamp(heat_output * 1e-2, 0, fuel_consumption)
```

Comment:
since the `fuel_consumption` was already multiplied with delta_time, having another delta_time was doubling the numbers and that caused the production to become infinite (and so we had the million credits shenanigans)